gr00t/experiment/data_config.py

```diff
@@ -880,6 +964,7 @@ def transform(self):
 ###########################################################################################
 
 DATA_CONFIG_MAP = {
+    "x1": X1DataConfig(),
```

Comment:
commented above. if not ready we can remove this for now. else i would name it as `agibot_x1`